app/models/entitlement.rb

```diff
@@ -56,8 +56,7 @@ def remove_tag_from_managed_filter(filter_to_remove)
       *category, _tag = filter_to_remove.split("/")
       category = category.join("/")
       self.filters["managed"].each do |filter|
-        next unless filter.first.starts_with?(category)
-        next unless filter.include?(filter_to_remove)
+        next unless filter.first.starts_with?(category) || filter.include?(filter_to_remove)
```

Comment:
@Jaganathancse Thanks for looking at this.  Sorry, but I prefer the original.  I like `unless` with a single condition.  I usually convert `unless` with an `||` to an `if` with reversed `!` logic if I must keep the conditional.

In this case also, these `next` are like a guard clause at the beginning of each iteration of the loop.  I like to keep guard clauses simple and separate.